astropy/coordinates/representation/geodetic.py

```diff
@@ -94,10 +96,93 @@ def from_cartesian(cls, cart):
         Converts 3D rectangular cartesian coordinates (assumed geocentric) to
         geodetic coordinates.
         """
+        # Compute geodetic/planetodetic angles
         lon, lat, height = erfa.gc2gde(
             cls._equatorial_radius, cls._flattening, cart.get_xyz(xyz_axis=-1)
         )
-        return cls(lon, lat, height, copy=False)
+        return cls(
```

Comment:
Nitpick, but could you get this back to the default format? Be sure to delete the final `,` after `copy=False`, otherwise `black` will break it over several lines again.